### ISSUE TITLE
Translation of `occurs` seems to be incorrect

### ISSUE THREAD

**Issue by ericsson49** [MiniZinc/libminizinc#731]
The following model
```
var opt int: v;
constraint occurs(v) /\ v == 1;
solve satisfy;
output ["v = \(v)"];
```
results in
```
v = -2147483646
```
which violates the `v == 1` constraint.

Corresponding FlatZinc code
```
var int: X_INTRODUCED_1_ ::var_is_introduced :: output_var;
var int: X_INTRODUCED_3_ ::var_is_introduced ;
constraint int_eq(X_INTRODUCED_3_,1);
solve  satisfy;
```
misses any constraint that would tie `X_INTRODUCED_1_` and `X_INTRODUCED_3_` values.

**Comment by cyderize** [comment]
I think this has already been fixed in the `develop` branch and will be included in the next release.

**Comment by cyderize** [comment]
This should now be fixed in 2.8.0
